### PR TITLE
Proofread second runtime file

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**
@Adam-Dupaski @MarkvanMents I've made some edits to the second file in the runtime batch. Please let me know any comments.

### CODE REVIEW

File: content/en/docs/refguide/runtime/runtime-server.md [hunk 1]
```diff
@@ -9,21 +9,21 @@ tags: ["runtime", "runtime server", "stateless", "database", "java", "microflows
...
 
 ## 2 Description
 
-The Runtime Server is deployed to the cloud (see [Runtime Deployment](/refguide/runtime-deployment/) for more information) and waits for requests from the Mendix Client or a call from another app or service. It processes the request and returns the requested data, plus any additional state information where appropriate. For details of how this communication takes place, see [Communication Patterns in the Mendix Runtime](/refguide/communication-patterns/). The Runtime Server itself is stateless, which allows it to be efficiently scaled to multiple instances.
+The Runtime Server is [deployed to the cloud](/refguide/runtime-deployment/) and waits for requests from the Mendix Client or a call from another app or service. It processes the request and returns the requested data, plus any additional state information where appropriate. For details of how this communication takes place, see [Communication Patterns in the Mendix Runtime](/refguide/communication-patterns/). The Runtime Server itself is stateless, which allows it to be efficiently scaled to multiple instances.
```
Comment (Adam-Dupaski) [inline content/en/docs/refguide/runtime/runtime-server.md]: Is this only "Mendix Cloud," or is it any cloud/deployment target?

`For details of how this communication takes place, see [Communication Patterns in the Mendix Runtime](/refguide/communication-patterns/).` - this cross-reference sits rather awkwardly as an entire full sentence in the middle of an paragraph full of info; can it be positioned better?

File: content/en/docs/refguide/runtime/runtime-server.md [hunk 2]
```diff
@@ -33,25 +33,25 @@ This is an interpreter written in Java and Scala, which uses the app model to de
 
 ### 2.3 Project Model
 
-This contains the model which defines how the app behaves, including domain models, microflows, import mappings etc. This is what the runtime core interprets to run the app.
+This contains the model which defines how the app behaves, including domain models, microflows, import mappings, and so on. This is what the runtime core interprets to run the app.
```
Comment (Adam-Dupaski) [inline content/en/docs/refguide/runtime/runtime-server.md:36]: "runtime core" is capitalized below and elsewhere in the doc. What should the standard be? Please decide with Mark, update this doc, update all other docs where this term is used, and add to PNG if capitalized.

File: content/en/docs/refguide/runtime/runtime-server.md [hunk 3]
```diff
@@ -63,15 +63,13 @@ This is a service which provides data to the app. A service could be another Men
 
 ### 2.10 Relational Database
 
-This is the database (or sometimes the schema of a shared database) which holds the objects as defined in the domain model(s) in the app.
+This is the database (or sometimes the schema of a shared database) which holds the objects as defined in the domain models in the app.
 
 ### 2.11 Query Executor
 
-This manages the CRUD (create, read, update, and delete) operations for retrieving and storing data in the relational database which is bound to the app. The operations are performed using SQL which is tailored to the underlying database.
+This manages the CRUD (create, read, update, and delete) operations for retrieving and storing data in the relational database which is bound to the app. The operations are performed using SQL which is tailored to the underlying database. Where queries are not formatted in SQL, the query executor converts them from their original format (for example XPath or OQL).
```
Comment (Adam-Dupaski) [inline content/en/docs/refguide/runtime/runtime-server.md]: Need a comma after "for example"

File: content/en/docs/refguide/runtime/runtime-server.md [hunk 4]
```diff
@@ -99,11 +97,11 @@ This receives requests from the Mendix Client, decodes them and passes them to t
 
 ### 2.18 Custom Request Handler
 
-This is a request handler added to the app using the com.mendix.core.Core#addRequestHandler(…) API call.
+This is a request handler added to the app using the `com.mendix.core.Core#addRequestHandler(…)` API call.
```
Comment (Adam-Dupaski) [inline content/en/docs/refguide/runtime/runtime-server.md]: What's the function of `(..)` here, is that clear to the reader?

File: content/en/docs/refguide/runtime/runtime-server.md [hunk 5]
```diff
@@ -99,11 +97,11 @@ This receives requests from the Mendix Client, decodes them and passes them to t
...
 
 ### 2.19 External Service Requests
 
-This receives requests from other services, decodes them and passes them to the Runtime Core or Object Manager, and formats a response to the service once the request has been processed. These requests can be:
+This receives requests from other services, decodes them, and passes them to the Runtime Core or Object Manager. After the request has been processed, this component formats a response to the request. The following requests can be processed:
```
Comment (Adam-Dupaski) [inline content/en/docs/refguide/runtime/runtime-server.md]: Same thing with "Object Manager" as with "Runtime Core" - what should the capitalization standard be? Please decide with Mark and make changes accordingly.

File: content/en/docs/refguide/runtime/runtime-server.md [hunk 6]
```diff
@@ -112,12 +110,12 @@ This receives requests from other services, decodes them and passes them to the
...
-This decodes HTTPS requests from Mendix Clients or other services and passes them to the Runtime Server.
+This decodes HTTPS requests from Mendix Clients or other services, and passes them to the Runtime Server.
 
 ### 2.21 Mendix Client
 
-This is the browser or device which allow the end-user to interact with the app. This can be a web browser, such as Chrome, or a mobile device, such as an iPhone. It typically has a screen, pointer device, and input device to allow end-users to use the app.
+This is the web browser (for example, Google Chrome) or mobile device (for example, an iPhone) which allows the user to interact with the app. It typically has a screen, pointer device, and input device to allow users to use the app.
 
-The Runtime Server communicates with the Mendix Client using a private API called *xas*.
+The Runtime Server communicates with the Mendix Client by using a private API called *xas*.
```
Comment (Adam-Dupaski) [inline content/en/docs/refguide/runtime/runtime-server.md]: Should "xas" be italicized? Also, if it's a private API, do we even need to name it?

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 1 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md]: @MarkvanMents about Adam's first question, I'm guessing any deployment target, correct?

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 2 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md:36]: @MarkvanMents is runtime core capitzalized?

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 3 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md]: Done.

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 4 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md]: @MarkvanMents I copied this one from the original, so I'm not sure. What does the (...) stand for?

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 5 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md]: @MarkvanMents let's discuss the capitalization of Object Manager.

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 6 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md]: For the italics, I was thinking of it as a term that is later defined, but I can remove them. Not sure about the name - @MarkvanMents what do you think?

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 2 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md:36]: not capitalized.

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 4 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md]: name of the request handler

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 5 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md]: not capitalized

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 1 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md]: Merged and simplified the link, changed "deployed to Cloud" to just "deployed".

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 2 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md:36]: Changed.

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 4 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md]: Updated.

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 5 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md]: Fixed.

File: content/en/docs/refguide/runtime/runtime-server.md (hunk 6 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/refguide/runtime/runtime-server.md]: Removed italics.

**Pull request merged**